### DISCUSSION TITLE
Is old firmware available ?

### DISCUSSION THREAD

**Discussion by UFO-OS9** [ZuluSCSI/ZuluSCSI-firmware#112] [Q&A] [answered]
Hello,
I have [zuluscusi RP2040 board](https://store.rabbitholecomputing.com/ZuluSCSI-V1-1-p/zuluscsi-rp2040.htm) currently with FW Version: 1.1.1-release Oct 25 2022  + 32Gb-2022 sandisk SD card 
I also have 2gb Blank image file (HD6.hda) & a working image file (HD6.imgc) from a colleague who assures me his setup works perfectly (not yet sure about his board or firmware versions though- waiting for confirmation).

Scsi drive requirements for this machine:  Scsi ID 6 with termination enabled

My problem is niether of these images work.
Correct me if I'm wrong, if I use the blank image, then the board should emulate a single scsi drive, whereby I can install the custom OS9 operating system & software via my 2 floppy diskettes, which is the normal process when replacing the original scsi  mechanical drive.
If the drive is an image how can you possibly write to the image, how can it be possibly formatted ? (logfile attached)

Secondly when I use a "proven working image"  nothing happens.

After trawling pages of manuals & discussions here, I cannot find any Firmware files other than a link to the version I already have:
FW Version: 1.1.1-release Oct 25 2022 , which I have also re-installed via USB C port.
So, if my colleague has a working version, he therefore must have different firmware.

Are there links to any "older" Firmware, to try ??  
Is a file extension of .imgc acceptable  ??

Any help would be most appreciated !

[zululog.txt](https://github.com/ZuluSCSI/ZuluSCSI-firmware/files/10148950/zululog.txt)

**Comment by PetteriAimonen** [comment, marked as answer]
Old releases are available to download here:
https://github.com/ZuluSCSI/ZuluSCSI-firmware/releases

**Reply by UFO-OS9** [reply]
Thanks for the link... It now transpires that my colleague has a totally different board....

So, it's back to basics. As I understand from a previous post:
"All you have to do is create an image file (it can be empty, or contain transplanted data), that's 4GB or smaller, on your SD card What you name the image file (HD0.img, HD0.hda, etc) doesn't particularly matter; Only that the file name starts with HD, and has a unique, valid SCSI ID of 0-7). From that point on, you need to treat the ZuluSCSI as you would any new, blank hard drive, from the perspective of the computer it's attached to."

I have [zuluscusi RP2040 board](https://store.rabbitholecomputing.com/ZuluSCSI-V1-1-p/zuluscsi-rp2040.htm)

I am missing the plot somewhere......  ! I have tried every conceivable option, can someone verify my process:
Format SD card Fat32 in a windows 10 pc
Create & place 2Gb blank image file named HD6.hda on SD card  (ID 6 termination enabled via dip switch)
remove SD card from windows PC
Put SD card into 2040 Board (no power cable)
install 2040 board  in old scsi driven OS9 computer unit

From here after nothing works... i cannot format my new drive using my tools floppy diskette
Do I need to set some parameters on an ini file ??  as log file returns text about sectors not being divisible

Any help would be appreciated
